gateway/src/apicast/policy/soap/policy.lua

```diff
@@ -0,0 +1,68 @@
+--- SOAP Policy
+-- This policy adds support for a very small subset of SOAP.
+-- This policy basically expects a SOAPAction URI in the SOAPAction header or
+-- the content-type header.
+-- The SOAPAction header is used in v1.1:
+-- https://www.w3.org/TR/2000/NOTE-SOAP-20000508/#_Toc478383528, whereas the
+-- Content-Type header is used in v1.2:
+-- https://www.w3.org/TR/soap12-part2/#ActionFeature
+-- The SOAPAction URI is matched against the mapping rules defined for the
+-- service and the usage resulting from that is authorized and reported
+-- against 3scale's backend.
```

Comment:
"...and reported to the API Management platform using the Service Management API" or similar?

Reply:
I usually refer to backend as '3scale's backend' sounds simpler than that, but would be good to reach an agreement on that. Should we call it by its _new upstream name_?

Reply:
I'd call it 3scale's backend.  Definitely try not to write its full name because I could not get it right and have to copy paste it from somehwere.

Reply:
Agreed @mikz